Week3/homework-midy/lottery.js

```diff
@@ -0,0 +1,35 @@
+function threeFive(startIndex, endIndex, three, five) {
+  const numbers = [];
+
+  const startParsed = parseInt(startIndex, 10);
+  const endParsed = parseInt(endIndex, 10);
+
+  const random = (start, end) => {
+    for (let i = 0; i < 6; i += 1) {
+      numbers.push(Math.floor(Math.random() * (end - start + 1) + start));
+    }
+    return numbers;
+  };
```

Comment:
Nice random number generator function. But you didn't need to generate random numbers according to the exercise. 

> The function should first generate an array containing values from start value to end value (inclusive).

So if you call it `threeFive(10, 15, sayThree, sayFive);` it should generate an array of `[10,11,12,13,14,15]`

Reply:
Oh sorry didn't notice that.